### ISSUE TITLE
valueOf leaves too inaccurate result

### ISSUE THREAD

**Issue by 837951602** [fasttime/Q-exact#3]
```
Q.pow(2,9e15+1).divide(Q.pow(5,3876089022660537)).valueOf()
2.718281828459045
Q.pow(2,9e15).divide(Q.pow(5,3876089022660537)).valueOf()
2.718281828459045
```
I understand that the log result is cut, but the document only says 
> The number returned by this function is subject to rounding: the result is not exact. Rationals with very large absolute value are represented as ±infinity, and very small rationals are rounded to ±0.

Directly say the error may go more than 50%?

**Comment by fasttime** [comment]
Agreed. I'll see how I can correct that part.

**Comment by fasttime** [comment]
Fixed per 1186cf635926025487cdf8ad7895e12d12a18309.
